### PR TITLE
Added 'Christmas' style board

### PR DESCRIPTION
**Pull request by joshfarrant**
Added a Christmas board style in the same way as the Halloween board was added.

`@<bot-name>: start xmas` to play.

### CODE REVIEW

File: src/connect4/emoji.js [hunk 1]
```diff
@@ -34,5 +34,17 @@ module.exports = {
...
         }
+    },
+    christmas: {
+        circle: {
+            red: ':santa:',
+            blue: ':christmas_tree:',
+            white: ':snowflake:'
+        },
+        celebrate: ':gift:',
+        fire: ':ribbon:',
```
Comment (wgpsutherland) [inline src/connect4/emoji.js]: I think this could be better as :snowman:

Comment (wgpsutherland) [inline src/connect4/emoji.js:45]: I think this could be better as :snowman:

Comment (joshfarrant) [inline src/connect4/emoji.js:45]: Which one in particular? The `fire` property?

Comment (wgpsutherland) [inline src/connect4/emoji.js:45]: Yeah. Sorry I thought the note had specified the line

Comment (joshfarrant) [inline src/connect4/emoji.js:45]: Done, that's updated in 620f778.

**Pull request merged**